code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -255,9 +247,9 @@
 	icon_state = "luxpen"
 	inhand_icon_state = "atropen"
 	base_icon_state = "luxpen"
-	volume = 60
-	amount_per_transfer_from_this = 60
-	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/c2/penthrite = 10, /datum/reagent/medicine/oxandrolone = 10, /datum/reagent/medicine/sal_acid = 10 ,/datum/reagent/medicine/omnizine = 10 ,/datum/reagent/medicine/leporazine = 10)
+	volume = 20
+	amount_per_transfer_from_this = 20
```

Comment:
```suggestion
	volume = 30
	amount_per_transfer_from_this = 30
```

I dont know if you plan on adding c2 chems here later, this is a pretty big nerf to a pen that costs 500 mining points more than the alternative